openslides/core/urls.py

```diff
@@ -7,4 +7,8 @@
     url(r'^core/url_patterns/$',
         views.UrlPatternsView.as_view(),
         name='core_url_patterns'),
+
+    # View for the projectors are handelt by angular.
+    url(r'^projector.*$',
```

Comment:
Can we move this to the global urls.py so that one can see the both entry points (/projector... and /...) at a glance?